### DISCUSSION TITLE
Website Deployment Help

### DISCUSSION THREAD

**Discussion by saiweekend** [community/community#113471] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

Hey Github Community -
I am fairly new to web dev, and I am having trouble with deploying my website. After I deployed it my CSSwas not loading, and neither was any of the images for my website. I am not sure why this is happening. 

I did some short hand research and found out that it may be due to Github's sensitive file path naming, so I went ahead and changed every path to include "../" if it was a sub folder or just remove the "/" from the beginning entirely. 

Unfortunately it still is not working and I have no one to really ask or where to go from here. Please help a hommie out! 

-saiweekend

Here is the link: https://github.com/saiweekend/Portfolio-Website

**Comment by SebastienXia** [comment]
## Hello, @saiweekend 

I also had this problem, but don't worry you can quickly fix it by deleting your browser **cache**. In your browsing history

`[close the thread and mark as answered if this helped you resolve your issue]`

**Reply by saiweekend** [reply]
Hmm. I just tried your solution, and it looks like my pages are still not loading correctly.

**Comment by SebastienXia** [comment]
Oh, I will see what happens in your code then.

**Comment by SebastienXia** [comment, marked as answer]
# @saiweekend 

Firstly your code is not well indented and not very nice to look at, I say this so that you can improve of course.

And secondly, the problem with the images was that you made a bad manipulation with the notion of path `../assets/img/{the image}` when you only had to remove a point `./asserts/img/{the image}`

**Reply by SebastienXia** [reply]
It's because you are fairly new to web dev 🤗 Keep going and learning @saiweekend

**Reply by saiweekend** [reply]
Hmm it is still not working for me. When I load up the code in VS-Code it works perfectly fine even with you suggestions and changes. Only when I try loading the page through Github it still is not loading properly.

**Reply by saiweekend** [reply]
Actually scratch that I actually managed to apply your solution and fix it. I think there was a delay for the changes. Thank you so much for the help!

**Comment by git-sumana** [comment]
Better if it is not a dynamic website, deploy it using Netlify and it even takes less time comparitively
